code/modules/mapping/mapping_helpers.dm

```diff
@@ -7,7 +7,7 @@
 	icon = 'icons/effects/mapping_helpers.dmi'
 	icon_state = ""
 
-	var/list/baseturf_to_replace
+	var/list/baseturf_to_replace = list(/turf/baseturf_bottom = TRUE)
```

Comment:
The default value is assigned in LateInitialize, not here.

Also by manual I meant map edits. It would probably be fine as a default baseturf if none are given by the looks of things though. This would need to be tested.

Reply:
as i see now we dont have another baseturfs. Only placeholder that places defoult Z lvl baseturf

Reply:
What we currently do or don't with this tool doesn't matter